src/services/apiService.ts

```diff
@@ -20,33 +24,82 @@ export class ApiService {
     }
 
     public postTransaction(transaction: Transaction): Promise<void> {
-        return new Promise(async (resolve, reject) => {
+        return new Promise(async (resolved, rejected) => {
             return Axios.post(this.getTransactionRoute(), transaction)
-                .then(() => resolve())
-                .catch((error) => reject(error));
+                .then(() => resolved())
+                .catch((error) => rejected(error));
         });
     }
 
+    public postDocumentToIPFS(base64EncodedPDF: string) {
+        return new Promise(async (solve, fail) => {
+            return Axios.post(this.getIPFSRoute(), {
+                base64EncodedPDF,
+            })
+                .then((response) => solve(response.data.hash))
+                .catch((error) => fail(error));
+        });
+    }
+
+    public getFeedbackForDocumentIPFSHash(hash: string) {
+        let feedback: Array<{ value: string; pubKey: string; time: number }> = [];
+        feedback_data.forEach((data) => {
+            if (data.hash === hash) {
+                feedback = data.feedback;
+            }
+        });
+        return feedback;
+    }
+
+    public addFeedbackInDocument(hash: string, feedback: string): void {
+        const dateTime = Date.now();
+        const timestamp = Math.floor(dateTime / 1000);
+        feedback_data.map((data) => {
+            if (data.hash === hash) {
+                data.feedback.push({ value: feedback, time: timestamp, pubKey: "PUBLICKEYOFUSER" });
+            }
+        });
+        fs.writeFile(
+            "./src/components/overview/feedback/feedback_mock/feedback.json",
+            JSON.stringify(feedback_data, null, 4),
+            (err) => {
+                if (err) {
+                    console.error(err);
+                    return;
+                }
+                console.log("File has been created");
```

Comment:
Best not to use console.log but the Blockr-Logger
```suggestion
                logger.info(JSON.stringify(feedback));
```